### DISCUSSION TITLE
Git restore command

### DISCUSSION THREAD

**Discussion by Ianneee** [NeogitOrg/neogit#1454] [Q&A]
Hi! 
Is `git restore <file>` implemented into neogit? I can't find it in help menu.

**Comment by CKolkey** [comment]
No, but you can reset any file thats been modified to any commit via the reset menu (`X`)

**Comment by windowsrefund** [comment]
Very new to this cool plugin and having just experienced `x` for myself, I noticed my original buffer didn't reload after closing the Neogit window. I had to manually reload the buffer with `:e`. Is this a known thing? Thanks in advance for any clarity.
